src/components/EasyPill/EasyPill.module.css

```diff
@@ -0,0 +1,4 @@
+.dropdownDeleteIcon {
+  align-self: center;
+  margin-right: var(--rvr-space-md);
+}
```

Comment:
Minor, but you should leave a blank newline at the end of the file.
It's a common thing with files under git version control, your editor likely has a setting for adding newlines at the end of files.